packages/dapp-toolkit/README.md

```diff
@@ -107,20 +108,22 @@ A signature produced by the wallet used to verify that the wallet is in control
 
 The signature is composed of:
 
-|                                       |                                                                                                                                                      |
-| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **prefix**                            | "R" (as in ROLA) in ascii encoding                                                                                                                   |
-| **challenge**                         | 32 random bytes provided by the dApp                                                                                                                 |
-| **length of dApp definition address** | String length of the dApp definition address                                                                                                         |
-| **dApp definition address**           | The dApp definition address of the requesting dApp                                                                                                   |
-| **origin**                            | The origin of the dApp (e.g. `https://dashboard.radixdlt.com`). This is a value that is added to the wallet data request by the Connector extension. |
+|                                       |                                                                |
+| ------------------------------------- | -------------------------------------------------------------- |
```

Comment:
It feels like we should have some column heading rather than just leaving these blank. **Item** and **Explanation** maybe?